### PR TITLE
Create delete_account.py

### PR DESCRIPTION
**Pull request by TAOTAO5**
add delete_account()

### CODE REVIEW

**Review by jonatan1609 — CHANGES_REQUESTED** [review]
In the function signature you can just do
`reason: str = ""`
So you could do
`reason=reason` instead of `reason=reason or ""`

and you also need to inherit DeleteAccount in https://github.com/pyrogram/pyrogram/blob/4ebdc39fdd885c6a9dd7289e474d28cbe972f609/pyrogram/methods/users/__init__.py

File: pyrogram/methods/users/delete_account.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#  along with Pyrogram.  If not, see <http://www.gnu.org/licenses/>.
+
+from pyrogram import raw
+from pyrogram.scaffold import Scaffold
+
+class DeleteAccount(Scaffold):
+    def delete_account(
+        self,
+        reason: str = None
+    ) -> bool:
```
Comment (AndrielFR) [inline pyrogram/methods/users/delete_account.py:26]: Use `reason: str = ""`.

File: pyrogram/methods/users/delete_account.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with Pyrogram.  If not, see <http://www.gnu.org/licenses/>.
+
+from pyrogram import raw
+from pyrogram.scaffold import Scaffold
+
+class DeleteAccount(Scaffold):
+    def delete_account(
+        self,
+        reason: str = None
```
Comment (AndrielFR) [inline pyrogram/methods/users/delete_account.py:25]: Use `reason: str = ""`.

Comment (TAOTAO5) [reply pyrogram/methods/users/delete_account.py]: ok👌, changes have been made

**Comment by jonatan1609** [comment]
👌

**Pull request closed without merging**